docs/content/filtering.md

```diff
@@ -17,7 +17,7 @@ Each path as it passes through rclone is matched against the include
 and exclude rules like `--include`, `--exclude`, `--include-from`,
 `--exclude-from`, `--filter`, or `--filter-from`. The simplest way to
 try them out is using the `ls` command, or `--dry-run` together with
-`-v`.
+`-v`. Filtering understand `-` as a file name to mean pipe from stdin.
```

Comment:
Can you expand this a bit more, eg `--filter-from`, `--exclude-from`, `--include-from`, `--files-from` understand `-` as a file name to mean read from standard input.